pkg/mermaidclient/Makefile

```diff
@@ -0,0 +1,10 @@
+all: gen/mermaid
```

Comment:
this needs to have a verify/update job in the top level makefile/CI - possibly just moved there